tools/es5to6-files-per-user.js

```diff
@@ -18,37 +18,37 @@ const filesPerHuman = (filesToCheck, humans) =>
     Math.max(filesToCheck.length / humans.length);
 
 const commercial = [
-    'janua',
-    'JonNorman',
-    'regiskuckaertz',
-    'rich-nguyen',
-    'katebee',
-    'Calum-Campbell',
-    'ScottPainterGNM',
+    'francis.carr@guardian.co.uk',
+    'jon.norman@guardian.co.uk',
+    'regis.kuckaertz@guardian.co.uk',
+    'rich.nguyen@guardian.co.uk',
+    'kate.whalen@guardian.co.uk',
+    'calum.campbell@guardian.co.uk',
```

Comment:
Thanks @regiskuckaertz. Perhaps we can move Calum to misc. I'm going to send a mass email around to gather recruits!